radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/tests/consensus/RandomLatencyTest.java

```diff
@@ -51,10 +52,12 @@
 			NetworkDroppers.bftSyncMessagesDropped()
 		)
 		.pacemakerTimeout(synchronousTimeout) // Since no syncing needed 6*MTT required
-		.checkConsensusLiveness("liveness", synchronousTimeout, TimeUnit.MILLISECONDS)
-		.checkConsensusSafety("safety")
-		.checkConsensusAllProposalsHaveDirectParents("directParents")
-		.checkConsensusNoTimeouts("noTimeouts");
+		.addTestModules(
+			ConsensusMonitors.safety(),
+			ConsensusMonitors.liveness(synchronousTimeout, TimeUnit.MILLISECONDS),
+			ConsensusMonitors.noTimeouts(),
+			ConsensusMonitors.directParents()
+		);
```

Comment:
How about renaming `addTestModules` method and using static imports? Then this part of code might look like
```
.withTest(safety(), liveness(synchronousTimeout, TimeUnit.MILLISECONDS), noTimeouts(), directParents));
```
This is somewhat more concise and more "fluent" for reading, as for me.

Reply:
I think this makes things too abstract for the sake of "fluent" reading. The module the monitor is testing is useful to know as each unit test tests specific modules (e.g. Consensus vs. Ledger).